### PR TITLE
Uber assembly at root project with content subprojects

### PR DESCRIPTION
**Pull request by pun-ky**

### CODE REVIEW

File: app/common/src/main/content/META-INF/vault/config.xml [hunk 1]
```diff
@@ -1,77 +0,0 @@
-<vaultfs version="1.1">
```
Comment (pun-ky) [inline app/common/src/main/content/META-INF/vault/config.xml]: this default will be provided / auto-generated by plugin

File: app/common/src/main/content/META-INF/vault/settings.xml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pun-ky) [inline app/common/src/main/content/META-INF/vault/settings.xml]: this also

File: settings.gradle [hunk 3]
```diff
@@ -1,7 +1,10 @@
 rootProject.name = 'example'
 
-include ':app'
```
Comment (pun-ky) [inline settings.gradle]: semicolon in the beginning not needed

File: gradle/buildscript.gradle [hunk 4]
```diff
@@ -23,7 +23,7 @@ repositories {
 }
 
 dependencies {
-    classpath 'com.cognifide.gradle:aem-plugin:1.2.+'
+    classpath 'com.cognifide.gradle:aem-plugin:1.2.1-SNAPSHOT'
```
Comment (piotr-wilczynski) [inline gradle/buildscript.gradle]: ?

Comment (pun-ky) [inline gradle/buildscript.gradle]: will be fixed during release

**Pull request merged**